projects/demo/src/modules/components/dialog/examples/9/index.html

```diff
@@ -1,11 +1,14 @@
 <div class="container">
     <tui-input-number
+        tuiHintContent="The amount must be less than 1000 ₽"
+        tuiHintDirection="top"
         tuiTextfieldPostfix="₽"
         tuiTextfieldSize="l"
         [formControl]="amountControl"
         [tuiTextfieldCleaner]="true"
         [tuiTextfieldLabelOutside]="true"
     >
+        0 ... 999 ₽
```

Comment:
do we need to add `maxlength` to this example?

Reply:
But where?

Reply:
```html
<tui-input-number
        tuiHintContent="The amount must be less than 1000 ₽"
        tuiHintDirection="top"
        tuiTextfieldPostfix="₽"
        tuiTextfieldSize="l"
        [formControl]="amountControl"
        [tuiTextfieldCleaner]="true"
        [tuiTextfieldLabelOutside]="true"
    >
        0 ... 999 ₽
        <input
            maxlength="3"  <------------ here
            inputmode="numeric"
            tuiTextfield
        />
    </tui-input-number>
```

Reply:
It doesn't work with maskito

Reply:
`maxlength="5"`, my mistake)